src/nextjournal/clerk/viewer.cljc

```diff
@@ -1386,9 +1386,9 @@
       (reduce compute-expanded-at state' value)
       state')))
 
-(defn collect-expandable-paths [state {:nextjournal/keys [value] :keys [path]}]
+(defn collect-expandable-paths [state {:nextjournal/keys [value presented?] :keys [path]}]
   (let [state' (assoc-in state [:expanded-at path] false)]
-    (if (vector? value)
+    (if (and (vector? value) (not presented?))
```

Comment:
As an alternative, we might just check that all (or just the first of) items in `value` are wrapped ones.